### ISSUE TITLE
Why App Secret is shared?

### ISSUE THREAD

**Issue by rafis** [blackgirl/Facebook_Oauth2_sample_for_extensions#1]
Here is your App Secret:

https://github.com/blackgirl/Facebook_Oauth2_sample_for_extensions/blob/cbfd85807f6ef498976c2789b92bf77cce63a8bd/index.js#L17

    var clientId = '1414994525410418'; // App ID
    var clientSecret = '0764de705582253ec90602c535fe8229'; // App Secret

According to [Facebook documentation](https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow) App Secret should be never shared and should be stored server-side only.

**Comment by sumiodz** [comment]
Hi .I am using this extension in developer mode when I click on sign in it will show the error
You are not logged in: You are not logged in. Please log in and try again


can you please tell me .I am using this extension for fb login .Please reply ASAP

**Comment by sergey-shpak** [comment]
@rafis you are right, I believe there should be a combination of client/server authorization for chrome extensions to keep secrets safe
